.github/workflows/release.yml

```diff
@@ -71,6 +71,20 @@ jobs:
                   GITHUB_TOKEN: ${{ secrets.KHAN_ACTIONS_BOT_TOKEN }}
                   NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+            - name: Send a Slack notification if a publish fails
+              if: failure()
+              uses: rtCamp/action-slack-notify@v2
+              env:
+                  SLACK_WEBHOOK: ${{ secrets.SLACK_EXERCISENEXT_WEBHOOK }}
+                  SLACK_CHANNEL: team-lc-perseus
+                  SLACK_MSG_AUTHOR: ${{ github.event.pull_request.user.login }}
+                  SLACK_USERNAME: GithubGoose
+                  SLACK_ICON_EMOJI: ":goose:"
+                  SLACK_MESSAGE: "Something went wrong with the release of ${{ github.event.repository.name }}. \nPlease check the logs → https://github.com/${{ github.repository }}/actions"
```

Comment:
**idea**: Are we able to get a link directly to this action run?  Looks like it might be possible to build using the `github.run_id`.  

This PR is an improvement overall so if that's not possible, I'm fine landing this PR as is.

Reply:
@jeremywiebe I will try! If we end up with a bad link we can fix it later